vowpalwabbit/options.cc

```diff
@@ -0,0 +1,15 @@
+#include "options.h"
```

Comment:
Is this worth a .cc file?

Reply:
Perhaps not? They'd need to be marked inline in the header

Reply:
I'm going to leave as is